persistent/interfaces.py

```diff
@@ -23,7 +23,7 @@
     from persistent.cPersistence import UPTODATE
     from persistent.cPersistence import CHANGED
     from persistent.cPersistence import STICKY
-except ImportError: #pragma NO COVER
+except ImportError: # pragma: no cover
```

Comment:
I hate to be a pain, but can you please back out the non-material / janitorial bits of this PR?  They make reviewing the "real" changes much harder.